mypy/messages.py

```diff
@@ -241,6 +241,12 @@ def format_simple(self, typ: Type, verbosity: int = 0) -> str:
           None -> None
           callable type -> "" (empty string)
         """
+        ret = self._format_simple(typ, verbosity)
+        if ret == "Module":
```

Comment:
Should really use 'Module' here. (Also I really detest the messages "Module does not blah" -- I always need to know the module name. But that's a separate concern.)